app/controllers/mail_templates_controller.rb

```diff
@@ -0,0 +1,27 @@
+class MailTemplatesController < ApplicationController
+  before_action :authorize
+  
+  def edit
+    @mail_template = MailTemplate.find_by(name: 'Invitation instructions')
+    @orphans = Organisation.not_null_email.null_users.without_matching_user_emails
+    render template: 'organisation_reports/without_users_edit',
+           layout: 'invitation_table'
+  end
+  
+  def update
+    @mail_template = MailTemplate.find(params[:id])
+    @orphans = Organisation.not_null_email.null_users.without_matching_user_emails
+    if @mail_template.update_attributes(mail_params)
+      flash[:notice] = 'Mail template has been updated'
+      redirect_to organisations_report_path
+    else
+      redirect_to :back
```

Comment:
is our acceptance test checking this "sad path" when the email template update fails?

We could possibly shorten this method like so:

```ruby
(redirect_to :back and return) unless @mail_template.update_attributes(mail_params)
flash[:notice] = 'Mail template has been updated'
redirect_to organisations_report_path
```

but we'd need the sad path coverage to check that ...

the `@mail_template = MailTemplate.find(params[:id])` feels like it could be extracted when we add other methods - we can probably safely ignore this "too many lines" comment for the moment as this controller will likely see other changes ...

I wonder if this would all make more sense as an ajax request and then we wouldn't need to pull in @orphans and so on - of course that would make the whole story more complex ...